File: IRepoProbaParticipant.java
package me.repository;

import me.model.Participant;
import me.model.Proba;
import me.model.ProbaParticipant;

import java.util.List;

public interface IRepoProbaParticipant extends Repository<String, ProbaParticipant> {
    int numarParticipantiProba(Integer p);
    List<ProbaParticipant> findProbaParticipantDupaProba(Integer p);
    List<ProbaParticipant> findProbaParticipantDupaParticipant(Integer p);
}

File: Concurs Inot C#/IRepoProbaParticipant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Problema_3_CSharp.me.model;

namespace Problema_3_CSharp.me.repository
{
    public interface IRepoProbaParticipant : IRepository<string,ProbaParticipant>
    {
        int NumarParticipantiProba(Proba p);
        IEnumerable<ProbaParticipant> FindProbaParticipantDupaProba(int p);
        IEnumerable<ProbaParticipant> FindProbaParticipantDupaParticipant(int p);
    }
}
